cert_issuer/blockchain_handlers/ethereum/connectors.py

```diff
@@ -60,13 +60,16 @@ class EtherscanBroadcaster(object):
     def __init__(self, base_url):
         self.base_url = base_url
 
-    def broadcast_tx(self, tx, api_token):
+    def broadcast_tx(self, tx, api_token):        
         tx_hex = tx
 
         broadcast_url = self.base_url + '?module=proxy&action=eth_sendRawTransaction'
         if api_token:
             '&apikey=%s' % api_token
         response = requests.post(broadcast_url, data={'hex': tx_hex})
+        if 'error' in response.json():
+            logging.error("Etherscan returned an error: %s", response.json()['error'])
+            raise BroadcastError()
```

Comment:
Could you have it broadcast `response.json()['error']` ?

Reply:
I've updated the PR. Does it match what you're looking for?

Reply:
Yup, thanks!